config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,8 @@
+CarrierWave.configure do |config|
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['AWS_KEY_ID'] || "",
+    aws_secret_access_key: ENV['AWS_KEY'] || "",
+  }
+  config.fog_directory  = ENV['AWS_BUCKET']
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.